model/common/src/icon4py/model/common/interpolation/interpolation_attributes.py

```diff
@@ -25,6 +25,11 @@
 POS_ON_TPLANE_E_X: Final[str] = "pos_on_tplane_e_x"
 POS_ON_TPLANE_E_Y: Final[str] = "pos_on_tplane_e_y"
 CELL_AW_VERTS: Final[str] = "cell_to_vertex_interpolation_factor_by_area_weighting"
+RBF_VEC_COEFF_C1: Final[str] = "rbf_interpolation_coefficient_cell_1"  # TODO: long name?
+RBF_VEC_COEFF_C2: Final[str] = "rbf_interpolation_coefficient_cell_2"  # TODO: long name?
```

Comment:
I _think_ these could be called (sem for vertices):
```suggestion
RBF_VEC_COEFF_C1: Final[str] = "rbf_interpolation_coefficient_cell_zonal"
RBF_VEC_COEFF_C2: Final[str] = "rbf_interpolation_coefficient_cell_meridional"
```
since the zonal/meridional components of the cell centers are used for the right-hand side of the linear solve. I don't know if `zonal` and `meridional` make sense physically in the names. Perhaps @vectorflux has an opinion on naming?

Reply:
Whoa, I take it back.  We have to make sure whether the components are really zonal/meridional, or rather tangential/normal.  I suspect they are the latter.

Reply:
Something like that, yeah. I'll keep the `1`/`2` suffixes, for lack of a (clearly) better alternative, and I don't want to cause extra confusion if zonal/meridional aren't precise enough. I'll remove the TODOs.